src/core/output-layout.cpp

```diff
@@ -1083,6 +1083,17 @@ class output_layout_t::impl
         LOGI("new output: ", output->name,
             " (\"", output->make, " ", output->model, " ", output->serial, "\")");
 
+        if (output->non_desktop)
+        {
+            LOGI("NON DESKTOP OUTPUT FOUND");
```

Comment:
These messages should be debug messages (LOGD) and please write them with normal text (e.g `Non-desktop output ...`) not all caps :)